docs/what/gms.md

```diff
@@ -2,6 +2,6 @@
 
 Metadata for [entities](entity.md) [onboarded](../how/entity-onboarding.md) to [GMA](gma.md) is served through microservices known as Generalized Metadata Service (GMS). GMS typically provides a [Rest.li](http://rest.li) API and must access the metadata using [GMA DAOs](../architecture/metadata-serving.md). 
 
-While a GMS is completely free to define its public APIs, we do provide a list of [resource base classes](../../metadata-restli-resource/src/main/java/com/linkedin/metadata/restli) to leverage for common patterns.
+While a GMS is completely free to define its public APIs, we do provide a list of [resource base classes](../../metadata-utils/src/main/java/com/linkedin/metadata/restli) to leverage for common patterns.
```

Comment:
Thanks for the contribution! Sorry for the confusion, the code is actually moved into a datahub-gma repo, the link should be
https://github.com/linkedin/datahub-gma/tree/master/restli-resources/src/main/java/com/linkedin/metadata/restli

Reply:
Thanks !! Updated the links per your comment.